projects/angular/src/forms/file-input/file-input-container.ts

```diff
@@ -128,7 +128,7 @@ export class ClrFileInputContainer extends ClrAbstractContainer {
   }
 
   protected get browseButtonDescribedBy() {
-    return `${this.label?.forAttr} ${this.fileInput.elementRef.nativeElement.getAttribute('aria-describedby')}`;
+    return `${this.label.labelId} ${this.fileInput.elementRef.nativeElement.getAttribute('aria-describedby')}`;
```

Comment:
The button needs to be described by the file input element, not the label. Please change this back.